lib/pengine/pe_output.c

```diff
@@ -558,10 +558,34 @@ int
 pe__cluster_options_text(pcmk__output_t *out, va_list args) {
     pe_working_set_t *data_set = va_arg(args, pe_working_set_t *);
 
+    out->list_item(out, NULL, "STONITH of failed nodes %s",
+                   is_set(data_set->flags, pe_flag_stonith_enabled) ? "enabled" : "disabled");
+
+    out->list_item(out, NULL, "Cluster is %s",
+                   is_set(data_set->flags, pe_flag_symmetric_cluster) ? "symmetric" : "asymmetric");
+
+    switch (data_set->no_quorum_policy) {
+        case no_quorum_freeze:
+            out->list_item(out, NULL, "No Quorum policy: Freeze resources");
+            break;
+
+        case no_quorum_stop:
+            out->list_item(out, NULL, "No Quorum policy: Stop ALL resources");
+            break;
+
+        case no_quorum_ignore:
+            out->list_item(out, NULL, "No Quorum policy: Ignore");
+            break;
+
+        case no_quorum_suicide:
+            out->list_item(out, NULL, "No Quorum policy: Suicide");
+            break;
+    }
+
     if (is_set(data_set->flags, pe_flag_maintenance_mode)) {
-        fprintf(out->dest, "\n              *** Resource management is DISABLED ***");
-        fprintf(out->dest, "\n  The cluster will not attempt to start, stop or recover services");
-        fprintf(out->dest, "\n");
+        out->list_item(out, NULL, "Resource management DISABLED (the cluster will not attempt to start, stop, or recover services)");
+    } else {
+        out->list_item(out, NULL, "Resource management enabled");
```

Comment:
I think we still want to handle pe_flag_maintenance_mode separately. I'm thinking a new mon_show_maintenance would be in the text defaults. It's probably worth keeping the existing output for text, i.e. the banner if it's set and nothing if it's unset.

Reply:
Sorry, I'm not quite following here.  Are you saying that in pe__cluster_options_text, I should first display the banner if pe_flag_maintenance_mode is set and then second, display the other information if mon_show_maintenance is set?

Reply:
I mixed two separate points:
* Maintenance mode should be shown by default in text mode, even though options as a whole are not. Similar to how dc can be shown by itself or as part of headers.
* When maintenance mode is shown in text mode, the banner should be shown if maintenance mode is set, and nothing should be shown if it's unset. I.e. it's treated specially compared to other options.